src/foam/nanos/controller/ApplicationController.js

```diff
@@ -752,7 +753,7 @@ foam.CLASS({
       }
 
       return new Promise(function(resolve, reject) {
-        self.stack.push(self.StackBlock.create({ view: { class: 'foam.u2.view.LoginView', mode_: 'SignIn' }, parent: self }));
+        self.stack.push(self.StackBlock.create({ view: { ...(self.loginVariables?.loginView ?? { class: 'foam.u2.view.LoginView' }), mode_: 'SignIn' }, parent: self }));
```

Comment:
Why are we doing this? loginVariables seems like another context? Can't we just register a replacement for the LoginView?

Reply:
loginView on theme can be a viewSpec so we need to pass the whole viewSpec not just the class